lightapi/rate_limiter.py

```
@@ -136,34 +140,35 @@ def is_rate_limited(
-            },
-            status_code=429,
-            headers={
-                "Retry-After": str(retry_after),
-                "X-RateLimit-Limit": str(limit),
-                "X-RateLimit-Remaining": "0",
-                "X-RateLimit-Reset": str(int(time.time() + reset_seconds)),
-            },
-        )
+
+def get_rate_limit_response(
+    self, request: Request, window: str = "minute"
```

Ruff (0.15.5)

[warning] Unused function argument: `request`

(ARG001)